[PATCH] fix build with script+dbus enabled, but no gui

The script/qtdbus plugin doesn't need gui support, so don't try to
link it against libQtGui.

Signed-off-by: Peter Korsgaard <jacmet@sunsite.dk>
---
 src/plugins/script/qtdbus/qtdbus.pro |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: qt-4.7.3/src/plugins/script/qtdbus/qtdbus.pro
===================================================================
--- qt-4.7.3.orig/src/plugins/script/qtdbus/qtdbus.pro
+++ qt-4.7.3/src/plugins/script/qtdbus/qtdbus.pro
@@ -1,6 +1,6 @@
 TARGET  = qtscriptdbus
 include(../../qpluginbase.pri)
-QT = core gui script
+QT = core script
 CONFIG += qdbus
 
 SOURCES += main.cpp
